Formula/opentsdb.rb

```diff
@@ -3,7 +3,7 @@ class Opentsdb < Formula
   homepage "http://opentsdb.net/"
   url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.4.0/opentsdb-2.4.0.tar.gz"
   sha256 "a2d6a34369612b3f91bf81bfab24ec573ab4118127dc1c0f0ed6fc57318d102c"
-  license "LGPL-2.1"
+  license "LGPL-2.1-or-later"
```

Comment:
https://github.com/OpenTSDB/opentsdb/blob/7b4994080ebf823042dda0de4d44da494c1d398c/src/core/AbstractQuery.java#L4-L12
> you can redistribute it and/or modify it
under the terms of the GNU Lesser General Public License as published by
the Free Software Foundation, either version 2.1 of the License, or (at your
option) any later version